.github/ci/files/bin/console

```diff
@@ -16,11 +16,13 @@
 ob_get_clean();
 
 include __DIR__ . "/../vendor/autoload.php";
-\Pimcore\Bootstrap::setProjectRoot();
 
+define('PIMCORE_PROJECT_ROOT', __DIR__ . '/..');
```

Comment:
That might be a problem if you're using custom directories (env variable). 
Why is this necessary, should be covered `\Pimcore\Bootstrap::setProjectRoot()` anyway, right?

Reply:
setProjectRoot assumes that it is in the vendor folder. When you use this console, you are in the pimcore only scope. I can path setProjectRoot, so that looks after pimcoreOnly/project scope if you prefer there

Reply:
Oh, my fault, I'm so sorry, it's only for CI Actions, so it's of course fine 😊